packages/buttons/src/styled/StyledButton.ts

```diff
@@ -49,7 +58,8 @@ export const getHeight = (props: IButtonProps & ThemeProps<DefaultTheme>) => {
  * 3. shifting :focus-visible from LVHFA order to preserve `color` on hover
  * 4. set default outline-color for smooth transition without artifacts
  */
-const colorStyles = (props: IButtonProps & ThemeProps<DefaultTheme>) => {
+const colorStyles = (props: IButtonProps & ThemeProps<IGardenTheme>) => {
+  const { shades } = props.theme.variables.buttons;
```

Comment:
Not married to shades being themeable this way - but I found it clunky and anti-thetical to how `getColor` works to define colors, with shades, in semantic variable generation. If we move that generation step into the component, this might be less of an issue.